WooCommerce/Classes/ViewRelated/Bookings/Booking Details/UpdateAttendanceStatusView.swift

```diff
@@ -83,14 +86,20 @@ private extension UpdateAttendanceStatusView.AttendanceStatus {
         case .booked:
             return "calendar.badge.checkmark"
         case .checkedIn:
-            return "calendar.and.person"
+            return "person.fill.checkmark"
```

Comment:
Just curious: where can I see this update in the design? I did not see this in the i2, maybe I missed the conversation somewhere.